javascript/javascript2/week1/danishLetters.js

```diff
@@ -0,0 +1,22 @@
+//Find and count the Danish letters
+///////////////////////////////////////////////
+function findDanishLetters (string) {
+    const danishLetters = ["æ", "ø", "å"]
+    const count = {total: 0}
+    for (const letter of string.split("")) {
+        if (danishLetters.includes(letter)) {
+            if (!count[letter]) {
+                count[letter]= 0
+            }
+            count.total++
+            count[letter]++
+        }
```

Comment:
Good solution!
Did you work on this with @Jul-S?

Reply:
thanks, no.